editor/styling/stylesheets/_dialogs.scss

```diff
@@ -295,28 +295,14 @@ $spacing-largest: 40px;
         @include icon-extension-template(-df-folder, -df-folder-active)
     }
 
-    &.resource-ext- {
-        /* Script files */
-        &fp, &gui_script, &lua,
-        &render_script, &script, &vp {
+    &.resource-ext {
+        &-script {
```

Comment:
It's not really a `resource-ext-` anymore. Should we rename the prefix to `resource-kind-` or something like that?

Reply:
Yep!